apps/store/src/components/ProductItem/ProductItem.tsx

```diff
@@ -106,6 +114,21 @@ const Header = styled.div({
   borderBottomColor: theme.colors.borderTranslucent2,
 })
 
+const HeaderContent = styled.div({
+  display: 'grid',
+  gridTemplateColumns: '1fr auto',
+  columnGap: theme.space.md,
+  alignItems: 'flex-start',
```

Comment:
nit. Since `grid` layout is being used:

```suggestion
  alignItems: 'start',
```

I remember some talk about unifying `align-items` values for `flex` and `grid` long time ago. Maybe it's already safe to use `align-items: start/end` instead of `flex-start/flex-end` now?! 🤔

Reply:
Seems like they are actually different but I have no idea how... 😅

start
The items are packed flush to each other toward the start edge of the alignment container in the appropriate axis.

flex-start
The cross-start margin edges of the flex items are flushed with the cross-start edge of the line.

https://developer.mozilla.org/en-US/docs/Web/CSS/align-items